classes/Device.class.php

```diff
@@ -1871,7 +1871,7 @@ function GetChildDevicePicture($parentDetails, $rear=false){
 						$slot->H=$ymax-$ymin;
 					}else{
 						// Last slot isn't defined so just error out
-						break;
+						return;
```

Comment:
I'm on 7.0.4 but I checked the 5 docs before I made the change (it didn't seem like break and return were interchangeable in that version either.)